apis/v1alpha1/gateway_types.go

```diff
@@ -424,7 +424,6 @@ type RouteBindingSelector struct {
 	//
 	// +optional
 	// +kubebuilder:default=networking.x-k8s.io
-	// +kubebuilder:validation:MinLength=1
 	// +kubebuilder:validation:MaxLength=253
 	Group string `json:"group,omitempty"`
```

Comment:
I was about to lgtm when I noticed that this field has `+optional`.
Why can't the user not specify a group at all? That would pass validation here, right?

Also, a related question, why is this not a pointer?